### DISCUSSION TITLE
Ignite chain build not building <project-name>d file

### DISCUSSION THREAD

**Discussion by aakash4dev** [cosmos/community#20] [Chain Builders (Cosmos SDK)]
![image](https://github.com/cosmos/cosmos-sdk/assets/94034414/ea4da0fd-8b22-48eb-9015-c7c8e109575a)

sometimes ignite chain build don't work, e.g.
```
aakash@Aakash:~/learn/cosm2$ ignite chain build
🗃  Installed. Use with: cosm2d
aakash@Aakash:~/learn/cosm2$ cosm2d
cosm2d: command not found
```

**Comment by julienrbrt** [comment]
Have you added `$GOPATH/bin` to your `$PATH`?
`$PATH=$PATH:$GOPATH/bin`
https://go.dev/doc/gopath_code#GOPATH
